sbn-nd/DAQInterface/configs/standard/ptb01PULL.fcl

```diff
@@ -0,0 +1,28 @@
+#include "ptb01.fcl"
+
+## Pull mode parameters
+    daq.fragment_receiver.request_address:        "227.128.11.129" # -- multicast request address
+    daq.fragment_receiver.multicast_interface_ip: "10.226.36.0" # -- should match the private net
+    daq.fragment_receiver.request_port: 3001   # UDP request port
+    daq.fragment_receiver.request_mode: window # pull mode buffer or window
+<<<<<<< HEAD
```

Comment:
Seems something didn't merge right

Reply:
@aaduszki  It looks like `daq.fragment_receiver.request_window_width` and `daq.fragment_receiver.request_window_offset` are defined twice for some reason. These two parameters aren't yet finalized so any of the 2 redundant parameters can be removed.

Reply:
Conflict resolved in this file.